src/libcommon/utility/utility.cpp

```diff
@@ -769,8 +769,8 @@ bool CommonUtility::dirNameIsValid(const SyncName &name) {
 
 // Check if dir name is valid by trying to create a tmp file
 bool CommonUtility::fileNameIsValid(const SyncName &name) {
-    if (!std::filesystem::exists(std::filesystem::temp_directory_path() / tmpDirName)) {
-        std::error_code ec;
+    std::error_code ec;
```

Comment:
You do not use `ec` after the next call. Why declaring it before then? It is to avoid that `std::filesystem::exists` throws an exception?